File: TrabajoI.C/JS/clase/ClaseC.js
class ClaseC {
    constructor(nombre, edad) {
        if (nombre && edad) {
            this.nombre = nombre;
            this.edad = edad;
            console.log(`Constructor sobrecargado: nombre y edad → ${nombre}, ${edad}`);
        } else if (nombre) {
            this.nombre = nombre;
            console.log(`Constructor sobrecargado: solo nombre → ${nombre}`);
        } else {
            console.log("Constructor sobrecargado: sin datos");
        }
    }
}

File: TrabajoI.C/C#/clase/ClaseC.cs
public class C : Figura
{
    private string nombre;
    private int edad;

    public C(string nombre)
    {
        this.nombre = nombre;
        Console.WriteLine($"C sobrecargado (solo nombre): {nombre}");
    }

    public C(string nombre, int edad)
    {
        this.nombre = nombre;
        this.edad = edad;
        Console.WriteLine($"C sobrecargado (nombre + edad): {nombre}, {edad}");
    }

    public override void Dibujar()
    {
        Console.WriteLine("Dibujando C");
    }
}
